behind-the-veil-siteroot/imports/ui/components/util.jsx

```diff
@@ -55,17 +57,33 @@ export function useUserInfo() {
                 (userInfo.profileImage ? (userInfo.profileImage.imageName !== fetchedProfileImage.imageName ||
                  userInfo.profileImage.imageSize !== fetchedProfileImage.imageSize) : fetchedProfileImage !== null)
             ) {
-                setUserInfo(
-                    {
-                        "id": fetchedUserId,
-                        "username": fetchedUsername,
-                        "email": fetchedEmail,
-                        "emailVerified": fetchedEmailVerified,
-                        "alias": fetchedAlias,
-                        "type": fetchedType,
-                        "profileImage": fetchedProfileImage,
-                    }
-                )
+                if (fetchedAlias == 'artist') {
```

Comment:
I think we should be checking for fetchedType right? Otherwise, only a user with the alias of artist will have access to artistServiceLocation and artistPaymentAccount

Reply:
yup, my bad. Fixed and pushed. Thankyou!